### PR TITLE
Cleanup/minor edits

### PR DESCRIPTION
**Pull request by davidlmobley**
Fixes a few references and some grammar, reorganizes slightly, resolves a couple to-do items.

Do a quick check, @JIMonroe ?

### CODE REVIEW

File: paper/basic_training.tex [hunk 1]
```diff
@@ -95,25 +100,35 @@ \section{Introduction}
...
+Furthermore, many systems of interest, such as polymers (biological and otherwise) have slow motions that must be captured for accurate calculation of properties.
+For example, for proteins, relevant timescales span from nanoseconds to seconds or more, and even rearrangements of buried amino acid sidechains can in some cases take microseconds or more, with larger conformational changes and protein folding taking even longer~\cite{Schlick:2010:, Mobley:2012:JComputAidedMolDes}. 
+Recent hardware innovations have made microsecond-length simulations for biological systems of 50-100,000 atoms relatively routine, and herculean efforts have pushed the longest simulations out past the millisecond range, but the field would like to reach even longer timescales, meaning that switching to a more detailed energy model is only done with some trepidation slower energy evaluations mean less time available for sampling.
```
Comment (JIMonroe) [inline paper/basic_training.tex]: This sentence sort of runs on.  Probably better to break up.

File: paper/basic_training.tex [hunk 2]
```diff
@@ -95,25 +100,35 @@ \section{Introduction}
...
+Recent hardware innovations have made microsecond-length simulations for biological systems of 50-100,000 atoms relatively routine, and herculean efforts have pushed the longest simulations out past the millisecond range, but the field would like to reach even longer timescales, meaning that switching to a more detailed energy model is only done with some trepidation slower energy evaluations mean less time available for sampling. 
+Thus the need for speed limits the use of quantum mechanical descriptions.
+
+Thus, for the rest of this document we will restrict ourselves to classical MD.
 
-For the rest of this document we will restrict ourselves to classical MD.
+One other important note is that, within classical molecular simulations, bond breaking and forming is generally not allowed (with notable exceptions such as reactive force fields),  meaning that the overall topology or chemistry of a system will remain constant as a function of time.
+That is, the particles comprising the system move around, but its chemical identity is a constant over the course of the simulation.
```
Comment (JIMonroe) [inline paper/basic_training.tex]: This is somewhat unclear.  You seem to refer to the "chemical identity" of the entire system, which to me seems a little ambiguous.  With stoichiometrically balanced chemical reactions, the overall atomic composition of the system could stay constant, but the molecular species would be different.  If we're talking mainly about making and breaking bonds, we should maybe say that the chemical identity (including overall charge, etc.) of each _molecule_ in the system is a constant in time.

**Comment by JIMonroe** [comment]
I'm going to merge this.  I had only a couple of minor suggestions above, which can be addressed in a later revision.

**Comment by davidlmobley** [comment]
Thanks, @JIMonroe ; those seem to the point; I'll see if I can fix soon.

**Pull request merged**